tests/foreman/rhai/test_rhai_client.py

```diff
@@ -15,69 +15,23 @@
-    def setUpClass(cls):  # noqa
-        super().setUpClass()
-        # Create a new organization with prefix 'insights'
-        org = entities.Organization(name='insights_{}'.format(gen_string('alpha', 6))).create()
+@pytest.mark.skip_if_open('BZ:1892405')
+def test_positive_connection_option(module_org, activation_key):
+    """Verify that '--test-connection' option for insights-client
+    client rpm tests the connection with the satellite server connection
+    with satellite server
 
-        # Upload manifest
-        with manifests.clone() as manifest:
-            upload_manifest(org.id, manifest.content)
+    :id: 167758c9-cbfa-4a81-9a11-27f88aaf9118
 
-        # Create activation key using default CV and library environment
-        activation_key = entities.ActivationKey(
-            auto_attach=True,
-            content_view=org.default_content_view.id,
-            environment=org.library.id,
-            name=gen_string('alpha'),
-            organization=org,
-        ).create()
-
-        # Walk through the list of subscriptions.
-        # Find the "Red Hat Employee Subscription" and attach it to the
-        # recently-created activation key.
-        for subs in entities.Subscription(organization=org).search():
-            if subs.name == DEFAULT_SUBSCRIPTION_NAME:
-                # 'quantity' must be 1, not subscription['quantity']. Greater
-                # values produce this error: "RuntimeError: Error: Only pools
-                # with multi-entitlement product subscriptions can be added to
-                # the activation key with a quantity greater than one."
-                activation_key.add_subscriptions(data={'quantity': 1, 'subscription_id': subs.id})
-                break
-        cls.org_label = org.label
-        cls.ak_name = activation_key.name
-        cls.org_name = org.name
-
-    def test_positive_connection_option(self):
-        """Verify that '--test-connection' option for redhat-access-insights
-        client rpm tests the connection with the satellite server connection
-        with satellite server
-
-        :id: 167758c9-cbfa-4a81-9a11-27f88aaf9118
-
-        :expectedresults: 'redhat-access-insights --test-connection' should
-            return zero on a successfully registered machine to RHAI service
-        """
-        with VirtualMachine(distro=DISTRO_RHEL6) as vm:
-            vm.configure_rhai_client(self.ak_name, self.org_label, DISTRO_RHEL6)
-            test_connection = vm.run('redhat-access-insights --test-connection')
-            self.logger.info(f'Return code for --test-connection {test_connection.return_code}')
-            self.assertEqual(
-                test_connection.return_code, 0, '--test-connection check was not successful'
-            )
+    :expectedresults: 'insights-client --test-connection' should
+        return zero on a successfully registered machine to RHAI service
+    """
+    with VirtualMachine(distro=DISTRO_RHEL7) as vm:
```

Comment:
Have you considered trying this with broker? (non-blocker)

Reply:
New tests require RHEL8, and it looks like we don't have that on SatLab. That would result in inconsistency, where some tests in single module use old libvirt provider, and some use broker.
So I would prefer to leave them on old infra for now.

Reply:
@mirekdlugosz are you available to complete the RHEL_8 work? Just need someone to pick it up and complete it. If you are free, I think that would take half a sprint and then this can be updated to use Satlab.

Reply:
@mirekdlugosz I'm now seeing RHEL 8.3 templates in Satlab, so you should be able to do the conversion now.